Frontend/CO.CDP.OrganisationApp.Tests/Pages/ShareInformation/ShareCodeDetailsTests.cs

```diff
@@ -109,7 +109,7 @@ public async Task OnGetDownload_ShouldRedirectToPageNotFound_WhenPdfResponseIsNu
         var shareCode = "HDJ2123F";
 
         _dataSharingApiClientMock.Setup(x => x.GetSharedDataFileAsync(shareCode))
-            .ReturnsAsync((WebApiClient.FileResponse?)null);
+            .ReturnsAsync((WebApiClient.FileResponse?)null!);
```

Comment:
As discussed - this shouldn't return null - it should perhaps be throwing a 404 ApiException?